src/store/modules/job/actions.ts

```diff
@@ -86,8 +87,36 @@ const actions: ActionTree<JobState, RootState> = {
           }
           jobs.map((job: any) => {
             job['statusDesc'] = this.state.util.statusDesc[job.statusId];
-          })          
-          commit(types.JOB_HISTORY_UPDATED, { jobs, total });
+          }) 
+
+          const typeDefinedJobs: Job = jobs.map((job: any) => {
+            return { 
```

Comment:
As we are returning an object we don't need a return statement here.

Reply:
yes sir. Fixed it.